components/feature/webcompat-reporter/src/test/java/mozilla/components/feature/webcompat/reporter/WebCompatReporterFeatureTest.kt

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.webcompat.reporter
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.concept.engine.Engine
+import mozilla.components.concept.engine.webextension.MessageHandler
+import mozilla.components.concept.engine.webextension.Port
+import mozilla.components.concept.engine.webextension.WebExtension
+import mozilla.components.support.test.any
+import mozilla.components.support.test.argumentCaptor
+import mozilla.components.support.test.eq
+import mozilla.components.support.test.mock
+import mozilla.components.support.webextensions.WebExtensionController
+import org.json.JSONObject
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+
+@RunWith(AndroidJUnit4::class)
+class WebCompatReporterFeatureTest {
+
+    @Before
+    fun setup() {
+        WebExtensionController.installedExtensions.clear()
+    }
+
+    @Test
+    fun `install installs the webextension`() {
```

Comment:
nit: `install installs`

Reply:
that actually was in purpose, as the method is called `install` and it `installs the webextension`. But yeah, that's weird, I'll make that `installs the webextension`.

Reply:
Ok, now I got it :)

Reply:
I changed it either way, that'll confuse a lot of folks otherwise. Thanks for the feedback!